mmdet/models/data_preprocessors/data_preprocessor.py

```diff
@@ -377,37 +377,60 @@ def __init__(self, data_preprocessor: ConfigType) -> None:
         super().__init__()
         self.data_preprocessor = MODELS.build(data_preprocessor)
 
-    def forward(
-        self,
-        data: dict,
-        training: bool = False
-    ) -> Tuple[Dict[str, torch.Tensor], Dict[str, Optional[list]]]:
+    def forward(self, data: dict, training: bool = False) -> dict:
         """Perform normalization、padding and bgr2rgb conversion based on
         ``BaseDataPreprocessor`` for multi-branch data.
 
         Args:
-            data (Sequence[dict]): data sampled from dataloader.
+            data (dict): Data sampled from dataloader.
             training (bool): Whether to enable training time augmentation.
 
         Returns:
-            Tuple[Dict[torch.Tensor], Dict[Optional[list]]]: Each tuple of
-            zip(dict, dict) is the data in the same format as the model input.
+            dict:
```

Comment:
We should be more specifc to indicate its formats, e.g., in semi-supervised learning, the format is like 
```
'inputs': {'sup': xxx, 'unsup': xxx}

```

Reply:
Can take this https://github.com/open-mmlab/mmdetection/blob/master/mmdet/datasets/custom.py#L24 as an example